design/spaces.go

```diff
@@ -58,9 +58,10 @@ var spaceOwnedBy = a.Type("SpaceOwnedBy", func() {
 
 var spaceAttributes = a.Type("SpaceAttributes", func() {
 	a.Attribute("name", d.String, "Name for the space", func() {
-		a.MaxLength(62) // maximum space name length is 62 characters
-		a.MinLength(1)  // minimum space name length is 1 characters
-		a.Pattern("^[^_|-].*")
+		a.MaxLength(62)                                             // maximum name length is 62 characters
+		a.MinLength(1)                                              // minimum name length is 1 characters
+		a.Pattern("^(?i)(h(o(m([^e]|$)|[^m]|$)|[^o]|$)|[^h_-]).*$") // Match _|-|home space name
```

Comment:
Thanks, makes sense @hectorj2f - 
( thinking out aloud - in future we might have more terms such as 'home' and should come up with a nice way of handling blacklists )

Good for now!